ingestion/src/metadata/utils/amundsen_helper.py

```diff
@@ -37,6 +37,15 @@
             }
         },
     },
+    "kafka": {
```

Comment:
if we are using this file for bot atlas and amundsen instead of calling it `amundsen_helper` maybe we can call the file `metadata_service_helper` or something!